### PR TITLE
Refines docs on creating action plugins

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
A little wordsmithing on the nice new example of how to create action plugins. 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

### CODE REVIEW

File: docs/docsite/rst/dev_guide/developing_plugins.rst [hunk 1]
```diff
@@ -79,7 +80,7 @@ Developing particular plugin types
 Action plugins
 --------------
 
-Action plugins are useful when you want to modify data that is provided to any module, or filter the data that is returned by the module.
+Action plugins modify the data that is provided to any module, or filter the data that is returned by a module.
```
Comment (bcoca) [inline docs/docsite/rst/dev_guide/developing_plugins.rst]: I would say jinja2 filters are useful for those cases, not really worth creating action plugins for that.

action plugins are useful when you need 'local' processing on controller, i.e get local file for `copy: src=`.

**Review by bcoca — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/dev_guide/developing_plugins.rst (hunk 1 above)
Comment (acozine) [reply docs/docsite/rst/dev_guide/developing_plugins.rst]: @bcoca does 46fc7aa7616aa273c87bae74560b4996dffc9e52 do a better job of describing the use case for action plugins?

**Review by bcoca — APPROVED** [review]

File: docs/docsite/rst/dev_guide/developing_plugins.rst (hunk 1 above)
Comment (bcoca) [reply docs/docsite/rst/dev_guide/developing_plugins.rst]: yep

**Review by Akasurde — APPROVED** [review]

File: docs/docsite/rst/dev_guide/developing_plugins.rst [hunk 2]
```diff
@@ -137,11 +138,10 @@ The following is an example showing how you can modify return data from Ansible'
             return dict(ansible_facts=dict(ret))
 
 
-Let us assume, you want to check time skewness between Ansible controller and remote machine then, we can have above code.
-Here, we are capturing date and time for remote machine using the ``setup`` module. Then we are calculating the time delta between the captured time and
-the time of Ansible controller. We are returning time delta in days, seconds and microseconds.
+This code checks the time on the controller, captures the date and time for remote machine using the ``setup`` module, and calculates the difference between the captured time and
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/developing_plugins.rst]:
```suggestion
This code checks the time on the controller, captures the date and time for the remote machine using the ``setup`` module, and calculates the difference between the captured time and
```

**Pull request merged**